sql/tests: TestRandomSyntaxSchemaChangeColumn failed

[sql/tests.TestRandomSyntaxSchemaChangeColumn failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2862755&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2862755&tab=artifacts#/) on [master @ 9a335c3b2eaac8a5b36ed4e4299be0bbb3d9f548](https://github.com/cockroachdb/cockroach/commits/9a335c3b2eaac8a5b36ed4e4299be0bbb3d9f548):

```
[14:52:40]
[post issues]
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         goroutine 8553553 [select, 2 minutes]:
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptcache.(*Cache).periodicallyRefreshProtectedtsCache(0xc01a8acbb0, 0x5bcca60, 0xc00b9329f0)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptcache/cache.go:160 +0x25b
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc007dda200, 0x5bcca60, 0xc00b9329f0, 0x0, 0xc017a32600)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
[14:52:40]
[post issues]         created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         goroutine 8553583 [select, 1 minutes]:
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/sql/catalog/lease.(*Manager).PeriodicallyRefreshSomeLeases.func1(0x5bcca60, 0xc00b9329f0)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/sql/catalog/lease/lease.go:1808 +0x285
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc007dda200, 0x5bcca60, 0xc00b9329f0, 0x0, 0xc017a32e50)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
[14:52:40]
[post issues]         created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         goroutine 8553324 [select, 5 minutes]:
[14:52:40]
[post issues]         google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc008ed7ac0)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:69 +0xc8
[14:52:40]
[post issues]         created by google.golang.org/grpc.newCCBalancerWrapper
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:60 +0x172
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         goroutine 8553286 [select, 5 minutes]:
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5bcca60, 0xc00b9329f0)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:105 +0x13d
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc007dda200, 0x5bcca60, 0xc00b9329f0, 0x0, 0xc006286c10)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
[14:52:40]
[post issues]         created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         goroutine 8553316 [chan receive, 5 minutes]:
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/server.(*Server).startServeUI.func1(0x5bcca60, 0xc00b9329f0)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2022 +0x53
[14:52:40]
[post issues]         github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc007dda200, 0x5bcca60, 0xc00b9329f0, 0x0, 0xc004556160)
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
[14:52:40]
[post issues]         created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
[14:52:40]
[post issues]         	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
[14:52:40]
[post issues]         
[14:52:40]
[post issues]     rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
[14:52:40]
[post issues]     rsg_test.go:755: Crash detected: 
[14:52:40]
[post issues]         ALTER TABLE ident.ident ADD IF NOT EXISTS SURVIVE INTERVAL DAY TO MINUTE [ ]
[14:52:40]
[post issues]         
[14:52:40]
[post issues]         Stack trace:
[14:52:40]
[post issues]         timeout: "ALTER TABLE ident.ident ADD IF NOT EXISTS SURVIVE INTERVAL DAY TO MINUTE [ ]". currently executing: map[ALTER TABLE ident.ident ADD IF NOT EXISTS SURVIVE INTERVAL DAY TO MINUTE [ ]:1]
[14:52:40]
[post issues]     rsg_test.go:764: 6923 executions, 1292 successful
[14:52:40]
[post issues]     rsg_test.go:401: -- test log scope end --
[14:52:40]
[post issues] test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn880718491
[14:52:40]
[post issues] --- FAIL: TestRandomSyntaxSchemaChangeColumn (357.99s)
[14:52:40]
[post issues] ```
[14:52:40]
[post issues] <details><summary>Reproduce</summary>
[14:52:40]
[post issues] <p>
[14:52:40]
[post issues] <p>To reproduce, try:
[14:52:40]
[post issues] ```bash
[14:52:40]
[post issues] make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
[14:52:40]
[post issues] ```
[14:52:40]
[post issues] </p>
[14:52:40]
[post issues] </p>
[14:52:40]
[post issues] </details>
[14:52:40]
[post issues] <details><summary>Same failure on other branches</summary>
[14:52:40]
[post issues] <p>
[14:52:40]
[post issues] - #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]</p>
[14:52:40]
[post issues] </details>
[14:52:40]
[post issues] [See this test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project)
[14:52:40]
[post issues] <sub>[Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
[14:52:40]
[post issues] ", Labels:["O-robot" "C-test-failure" "branch-master"], Assignee:"mjibson"}: POST https://api.github.com/repos/cockroachdb/cockroach/issues: 422 Validation Failed [{Resource:Issue Field:assignee Code:invalid Message:}]
[14:52:40]
[Run Random Syntax tests] artifacts (7s)
[14:52:47]
[Run Random Syntax tests] exit status
[14:52:47]
[Run Random Syntax tests] Process exited with code 2
[14:52:47]
[Step 2/2] Process exited with code 2 (Step: Command Line)
[14:52:47]
[Step 2/2] Step Command Line failed
[14:52:47]
Publishing internal artifacts (4s)
[14:52:47]
Publishing artifacts (5s)
[14:52:53]
Build finished
```

cc @ajwerner

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878584&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878584&tab=artifacts#/) on master @ [7495d19e50d8fd14140220f3ba5c7ad951abafa6](https://github.com/cockroachdb/cockroach/commits/7495d19e50d8fd14140220f3ba5c7ad951abafa6):


```
        goroutine 8914744 [select]:
        google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc00983ce60, 0x1, 0x0, 0x0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/controlbuf.go:395 +0x125
        google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc01434b500, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/controlbuf.go:515 +0x1d3
        google.golang.org/grpc/internal/transport.newHTTP2Client.func3(0xc00f7fda40)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/http2_client.go:356 +0x7b
        created by google.golang.org/grpc/internal/transport.newHTTP2Client
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/http2_client.go:354 +0x123c
        
        goroutine 8908392 [select]:
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).Start.func8(0x5bddd00, 0xc000074120)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/registry.go:727 +0x20d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc011432200, 0x5bddd00, 0xc000074120, 0x0, 0xc013cf9e90)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8978568 [chan send, 1 minutes]:
        github.com/lib/pq.(*conn).watchCancel.func1(0xc0066e8240, 0xc00e1a5b80, 0xc00c457b00)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:104 +0x185
        created by github.com/lib/pq.(*conn).watchCancel
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:93 +0xa5
        
        goroutine 8908331 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*baseQueue).addToPurgatoryLocked.func2(0x5bddd80, 0xc013c16990)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/queue.go:1152 +0x346
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc011432200, 0x5bddd80, 0xc013c16990, 0x0, 0xc021940a40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8908120 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5bddd80, 0xc0116e6600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:105 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc011432200, 0x5bddd80, 0xc0116e6600, 0x0, 0xc0057dafe0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN ident INTERVAL CREATE IF NOT EXISTS FAMILY ident
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN ident INTERVAL CREATE IF NOT EXISTS FAMILY ident". currently executing: map[ALTER TABLE ident.ident ADD COLUMN ident INTERVAL CREATE IF NOT EXISTS FAMILY ident:1]
    rsg_test.go:764: 7997 executions, 1473 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn788688690
--- FAIL: TestRandomSyntaxSchemaChangeColumn (355.99s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891681&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891681&tab=artifacts#/) on master @ [e29d694755e4800f07df139137c71cf741829dd0](https://github.com/cockroachdb/cockroach/commits/e29d694755e4800f07df139137c71cf741829dd0):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8980127 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc00d3b8590, 0x418d)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc00d3b8580)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc022510320, 0x5c150c0, 0xc019bd9590)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc022a0e200, 0x5c150c0, 0xc019bd9590, 0x0, 0xc006825b40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9062374 [select]:
        github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher.(*RequestBatcher).Send(0xc000fc4000, 0x5c15040, 0xc000074120, 0xb, 0x5d51240, 0xc00e13c3c0, 0x0, 0x1, 0x0, 0x7600000000000000)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher/batcher.go:248 +0x1fc
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).gcTxnRecord(0xc026e22680, 0x5c15040, 0xc000074120, 0xb, 0xc0037aefc0, 0xce6f8a95823c9f9a, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:716 +0x35e
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2(0x5c15040, 0xc000074120)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:752 +0x68
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc022a0e200, 0x5c15040, 0xc000074120, 0x0, 0xc00eff4330)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8980213 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).RangeFeed.func1(0x5c15000, 0xc02f1bc580, 0x100000001, 0x70000c034321c20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go:66 +0x14d
        github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1(0xc022a0e200, 0x5495998)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166 +0x3a
        golang.org/x/sync/errgroup.(*Group).Go.func1(0xc02901a1b0, 0xc02901a1e0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57 +0x59
        created by golang.org/x/sync/errgroup.(*Group).Go
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:54 +0x66
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP IF EXISTS AFTER CASCADE
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP IF EXISTS AFTER CASCADE". currently executing: map[ALTER TABLE ident.ident DROP IF EXISTS AFTER CASCADE:1]
    rsg_test.go:764: 9250 executions, 1682 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn603151278
--- FAIL: TestRandomSyntaxSchemaChangeColumn (350.59s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891761&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891761&tab=artifacts#/) on master @ [e29d694755e4800f07df139137c71cf741829dd0](https://github.com/cockroachdb/cockroach/commits/e29d694755e4800f07df139137c71cf741829dd0):


```
        goroutine 8251892 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5c150c0, 0xc0134e0660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:105 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc009cdc380, 0x5c150c0, 0xc0134e0660, 0x0, 0xc01882eee0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8251891 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5c150c0, 0xc0134e0660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:105 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc009cdc380, 0x5c150c0, 0xc0134e0660, 0x0, 0xc01882eed0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8251902 [chan receive, 5 minutes]:
        github.com/cockroachdb/cmux.muxListener.Accept(...)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:184
        crypto/tls.(*listener).Accept(0xc004242240, 0x4a9c380, 0xc01022c390, 0x464d140, 0x7d001c0)
        	/usr/local/go/src/crypto/tls/tls.go:67 +0x37
        net/http.(*Server).Serve(0xc024ad0e00, 0x5bce480, 0xc004242240, 0x0, 0x0)
        	/usr/local/go/src/net/http/server.go:2937 +0x266
        github.com/cockroachdb/cockroach/pkg/server.(*Server).startServeUI.func4(0x5c150c0, 0xc0134e0660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2070 +0x3f
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc009cdc380, 0x5c150c0, 0xc0134e0660, 0x0, 0xc004242260)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8251896 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).processEvents(0xc000983450, 0x5c15000, 0xc00850d600, 0xc0074ae360, 0xc00e4ae060, 0xc00e4ae0c0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:303 +0x129
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).run(0xc000983450, 0x5c15000, 0xc00850d600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:214 +0x6e5
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc009cdc380, 0x5c15000, 0xc00850d600, 0x0, 0xc01882ef70)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER COLUMN LEAST SET NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER COLUMN LEAST SET NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER COLUMN LEAST SET NOT NULL:1]
    rsg_test.go:764: 5942 executions, 1091 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn392923736
--- FAIL: TestRandomSyntaxSchemaChangeColumn (349.67s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2894000&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2894000&tab=artifacts#/) on master @ [3aad3f8a372090ddaedad0ad58a945fb399ddbc6](https://github.com/cockroachdb/cockroach/commits/3aad3f8a372090ddaedad0ad58a945fb399ddbc6):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00bf4c280, 0x5c167a0, 0xc03b617050, 0x0, 0xc00261fd50)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8688715 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5c167a0, 0xc03a4e0600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:105 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00bf4c280, 0x5c167a0, 0xc03a4e0600, 0x0, 0xc00eace630)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8688707 [chan receive, 5 minutes]:
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop.func1(0x5c167a0, 0xc0328c0330)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:176 +0x76
        runtime/pprof.Do(0x5c167a0, 0xc0328c0330, 0xc0003415a0, 0x1, 0x1, 0xc005a16fb8)
        	/usr/local/go/src/runtime/pprof/runtime.go:40 +0xcc
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop(0xc004e4f000)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:175 +0x89
        created by github.com/cockroachdb/pebble.(*tableCacheShard).init
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
        goroutine 8688753 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*baseQueue).processLoop.func2(0x5c167a0, 0xc03b616c60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/queue.go:818 +0x15f
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00bf4c280, 0x5c167a0, 0xc03b616c60, 0x0, 0xc02fe46b40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8688734 [chan receive, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/server.(*Server).startServeUI.func1(0x5c167a0, 0xc03a4e0600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2022 +0x53
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00bf4c280, 0x5c167a0, 0xc03a4e0600, 0x0, 0xc021999de0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER VIRTUAL SET NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER VIRTUAL SET NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER VIRTUAL SET NOT NULL:1]
    rsg_test.go:764: 9204 executions, 1827 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn347323964
--- FAIL: TestRandomSyntaxSchemaChangeColumn (347.35s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898498&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898498&tab=artifacts#/) on master @ [4dc05cceea254854317a9374a4b88df06c0946a6](https://github.com/cockroachdb/cockroach/commits/4dc05cceea254854317a9374a4b88df06c0946a6):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_raft.go:649 +0x265
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00e406200, 0x5c19900, 0xc0114a1ad0, 0x0, 0xc00fa056c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8184341 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc007714350, 0x2fff)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc007714340)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc00be380a0, 0x5c19900, 0xc0114a1ad0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00e406200, 0x5c19900, 0xc0114a1ad0, 0x0, 0xc00fa052b0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8184398 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc007714350, 0x2ffe)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc007714340)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc00be380a0, 0x5c19900, 0xc0114a1ad0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00e406200, 0x5c19900, 0xc0114a1ad0, 0x0, 0xc00fa05640)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8190407 [select]:
        google.golang.org/grpc/internal/transport.(*http2Client).keepalive(0xc01c90d180)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/http2_client.go:1362 +0x193
        created by google.golang.org/grpc/internal/transport.newHTTP2Client
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/http2_client.go:305 +0x1a3c
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS OIDS INTERVAL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS OIDS INTERVAL". currently executing: map[ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS OIDS INTERVAL:1]
    rsg_test.go:764: 5979 executions, 1154 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn274973818
--- FAIL: TestRandomSyntaxSchemaChangeColumn (354.76s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907569&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907569&tab=artifacts#/) on master @ [66a1657247f8e1cd84ce22a32321fda6352d789f](https://github.com/cockroachdb/cockroach/commits/66a1657247f8e1cd84ce22a32321fda6352d789f):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8204139 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/security.(*CertificateManager).RegisterSignalHandler.func1(0xc01b5f2300, 0x5c40720, 0xc000074120, 0xc01b032000)
        	/go/src/github.com/cockroachdb/cockroach/pkg/security/certificate_manager.go:235 +0x159
        created by github.com/cockroachdb/cockroach/pkg/security.(*CertificateManager).RegisterSignalHandler
        	/go/src/github.com/cockroachdb/cockroach/pkg/security/certificate_manager.go:232 +0x65
        
        goroutine 8204141 [chan receive, 5 minutes]:
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop.func1(0x5c407a0, 0xc003b7a3f0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:176 +0x76
        runtime/pprof.Do(0x5c407a0, 0xc003b7a3f0, 0xc0005a47a0, 0x1, 0x1, 0xc0119057b8)
        	/usr/local/go/src/runtime/pprof/runtime.go:40 +0xcc
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop(0xc012d78e00)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:175 +0x89
        created by github.com/cockroachdb/pebble.(*tableCacheShard).init
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
        goroutine 8204167 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).processEvents(0xc012b6a340, 0x5c406e0, 0xc02810de00, 0xc00b1200f0, 0xc019978060, 0xc0199780c0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:303 +0x129
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).run(0xc012b6a340, 0x5c406e0, 0xc02810de00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:214 +0x6e5
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc01b5f2300, 0x5c406e0, 0xc02810de00, 0x0, 0xc01d352fd0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8204355 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*registration).outputLoop(0xc023fc7ea0, 0x5c406e0, 0xc023494940, 0xc023494940, 0xc01d353fa0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/registry.go:254 +0x1f9
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*registration).runOutputLoop(0xc023fc7ea0, 0x5c406e0, 0xc023494940)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/registry.go:271 +0xc5
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).run.func1(0x5c406e0, 0xc023494800)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:259 +0x65
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc01b5f2300, 0x5c406e0, 0xc023494800, 0x0, 0xc013615920)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER GOAL DROP NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER GOAL DROP NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER GOAL DROP NOT NULL:1]
    rsg_test.go:764: 9985 executions, 1881 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn906812222
--- FAIL: TestRandomSyntaxSchemaChangeColumn (352.83s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2913126&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2913126&tab=artifacts#/) on master @ [5f40d691ef9caf910f97fbad53802000c0ca72eb](https://github.com/cockroachdb/cockroach/commits/5f40d691ef9caf910f97fbad53802000c0ca72eb):


```
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc021c041d0, 0x255e)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc021c041c0)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0211dc140, 0x5d07440, 0xc0173b2db0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc023b8c300, 0x5d07440, 0xc0173b2db0, 0x0, 0xc0143f5ca0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8076634 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc021c041d0, 0x257a)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc021c041c0)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0211dc140, 0x5d07440, 0xc0173b2db0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc023b8c300, 0x5d07440, 0xc0173b2db0, 0x0, 0xc0143f5e90)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8083655 [select, 5 minutes]:
        google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc01ef53f40)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:69 +0xc8
        created by google.golang.org/grpc.newCCBalancerWrapper
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:60 +0x172
        
        goroutine 8076723 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*Server).PeriodicallyClearSQLStats.func1(0x5d07440, 0xc0169586c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:757 +0x265
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc023b8c300, 0x5d07440, 0xc0169586c0, 0x0, 0xc004b782a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP STRING
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP STRING". currently executing: map[ALTER TABLE ident.ident DROP STRING:1]
    rsg_test.go:764: 4276 executions, 770 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn419597926
--- FAIL: TestRandomSyntaxSchemaChangeColumn (359.79s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917181&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917181&tab=artifacts#/) on master @ [9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675](https://github.com/cockroachdb/cockroach/commits/9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675):


```
        	/usr/local/go/src/net/http/server.go:2937 +0x266
        github.com/cockroachdb/cockroach/pkg/server.(*Server).startServeUI.func4(0x5d11280, 0xc01c8ba7b0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2071 +0x3f
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f010200, 0x5d11280, 0xc01c8ba7b0, 0x0, 0xc014669d80)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8435279 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc0134440d0, 0x3065)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc0134440c0)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0017d3220, 0x5d11280, 0xc004843230)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f010200, 0x5d11280, 0xc004843230, 0x0, 0xc003c79d70)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8435200 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5d11280, 0xc01c8ba7b0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f010200, 0x5d11280, 0xc01c8ba7b0, 0x0, 0xc00980ecb0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8435178 [chan receive, 5 minutes]:
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop.func1(0x5d11280, 0xc011b8e210)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:176 +0x76
        runtime/pprof.Do(0x5d11280, 0xc011b8e210, 0xc0003415c0, 0x1, 0x1, 0xc0316d3fb8)
        	/usr/local/go/src/runtime/pprof/runtime.go:40 +0xcc
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop(0xc004ef0500)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:175 +0x89
        created by github.com/cockroachdb/pebble.(*tableCacheShard).init
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD IF NOT EXISTS RESUME INTERVAL DAY ARRAY
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD IF NOT EXISTS RESUME INTERVAL DAY ARRAY". currently executing: map[ALTER TABLE ident.ident ADD IF NOT EXISTS RESUME INTERVAL DAY ARRAY:1]
    rsg_test.go:764: 5835 executions, 1076 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn317778154
--- FAIL: TestRandomSyntaxSchemaChangeColumn (357.60s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920166&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920166&tab=artifacts#/) on master @ [3cd29547884feb24b150b67ca2a88073798a35fd](https://github.com/cockroachdb/cockroach/commits/3cd29547884feb24b150b67ca2a88073798a35fd):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/queue.go:818 +0x15f
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00ce4a300, 0x5ca5ea0, 0xc00b7c6f90, 0x0, 0xc060f46100)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9022981 [select]:
        github.com/cockroachdb/cockroach/pkg/util/retry.(*Retry).Next(0xc0026cb350, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:128 +0x187
        github.com/cockroachdb/cockroach/pkg/sql.schemaChangeResumer.Resume.func1(0x7100000035, 0xc000000000, 0xc0173b5280, 0x5d0e3c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:2153 +0x5f3
        github.com/cockroachdb/cockroach/pkg/sql.schemaChangeResumer.Resume(0xc00d64bb80, 0x5ca5ea0, 0xc015773ad0, 0x4aac980, 0xc0149e6400, 0xc00d64bb80, 0xc0026cb830)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:2274 +0x803
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func1(0xc054771e00, 0x5c4d860, 0xc0149e63f0, 0x5ca5ea0, 0xc015773ad0, 0x4aac980, 0xc0149e6400, 0xc0026cba60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1150 +0xae
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc002f9a4e0, 0x5ca5ea0, 0xc0157739e0, 0x4aac980, 0xc0149e6400, 0x5c4d860, 0xc0149e63f0, 0xc00d64bb80, 0xc00d0cebd0, 0x7, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1151 +0x877
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc002f9a4e0, 0x5ca5de0, 0xc002a32280, 0x5c4d860, 0xc0149e63f0, 0xc00d64bb80, 0xc00d0cebd0, 0x7, 0xc0092274a0, 0x16, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:255 +0x378
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1(0x5ca5de0, 0xc000c5b0c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:214 +0x165
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00ce4a300, 0x5ca5de0, 0xc000c5b0c0, 0x0, 0xc05be280a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8973730 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc012ce8250, 0x23f2)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc012ce8240)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc00f568140, 0x5ca5ea0, 0xc00b7c73e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00ce4a300, 0x5ca5ea0, 0xc00b7c73e0, 0x0, 0xc0409799d0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD IF NOT EXISTS VOTERS BIT ( 442 ) ARRAY
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD IF NOT EXISTS VOTERS BIT ( 442 ) ARRAY". currently executing: map[ALTER TABLE ident.ident ADD IF NOT EXISTS VOTERS BIT ( 442 ) ARRAY:1]
    rsg_test.go:764: 4448 executions, 825 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn690796522
--- FAIL: TestRandomSyntaxSchemaChangeColumn (356.78s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921793&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921793&tab=artifacts#/) on master @ [2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4](https://github.com/cockroachdb/cockroach/commits/2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:128 +0x187
        github.com/cockroachdb/cockroach/pkg/sql.schemaChangeResumer.Resume.func1(0x8d00000035, 0xc000000000, 0xc00d67d100, 0x5d0e540)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:2159 +0x5f3
        github.com/cockroachdb/cockroach/pkg/sql.schemaChangeResumer.Resume(0xc015e69e00, 0x5ca6020, 0xc006e40f00, 0x4aaca40, 0xc00ef7a948, 0xc015e69e00, 0xc0012af830)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:2280 +0x803
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func1(0xc006a28200, 0x5c4d9e0, 0xc00ef7a938, 0x5ca6020, 0xc006e40f00, 0x4aaca40, 0xc00ef7a948, 0xc002b13a60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1150 +0xae
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc000f43860, 0x5ca6020, 0xc006e40e40, 0x4aaca40, 0xc00ef7a948, 0x5c4d9e0, 0xc00ef7a938, 0xc015e69e00, 0xc01d66d840, 0x7, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1151 +0x877
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc000f43860, 0x5ca5f60, 0xc029e0a3c0, 0x5c4d9e0, 0xc00ef7a938, 0xc015e69e00, 0xc01d66d840, 0x7, 0xc003f72a20, 0x16, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:255 +0x378
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1(0x5ca5f60, 0xc00bc34f00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:214 +0x165
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc024946300, 0x5ca5f60, 0xc00bc34f00, 0x0, 0xc00f3663c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8611774 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5ca6020, 0xc02d57a660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc024946300, 0x5ca6020, 0xc02d57a660, 0x0, 0xc009271b40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8611825 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc00bff20d0, 0x2e02)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc00bff20c0)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc00074d7c0, 0x5ca6020, 0xc01a9c76e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc024946300, 0x5ca6020, 0xc01a9c76e0, 0x0, 0xc01afebc20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL:1]
    rsg_test.go:764: 5247 executions, 998 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn324191524
--- FAIL: TestRandomSyntaxSchemaChangeColumn (360.02s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924710&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924710&tab=artifacts#/) on master @ [2dbd33ce319439bc424a52896a03e0a7ccff07f1](https://github.com/cockroachdb/cockroach/commits/2dbd33ce319439bc424a52896a03e0a7ccff07f1):


```
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57 +0x59
        created by golang.org/x/sync/errgroup.(*Group).Go
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:54 +0x66
        
        goroutine 8551264 [select, 5 minutes]:
        google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc01b065c40)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:69 +0xc8
        created by google.golang.org/grpc.newCCBalancerWrapper
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:60 +0x172
        
        goroutine 8544926 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc02c12c550, 0x2a90)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc02c12c540)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc03cdea3c0, 0x5ca6220, 0xc0080e19e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc033be8200, 0x5ca6220, 0xc0080e19e0, 0x0, 0xc01bfc1c50)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8544880 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5ca6220, 0xc017950720)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc033be8200, 0x5ca6220, 0xc017950720, 0x0, 0xc00622ab20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8545066 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/stmtdiagnostics.(*Registry).poll(0xc01e4aeae0, 0x5ca6160, 0xc00f262c00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/stmtdiagnostics/statement_diagnostics.go:147 +0x2e5
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc033be8200, 0x5ca6160, 0xc00f262c00, 0x0, 0xc0032cc860)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident STRING ( 359 ) ARRAY [ 132 ]
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident STRING ( 359 ) ARRAY [ 132 ]". currently executing: map[ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident STRING ( 359 ) ARRAY [ 132 ]:1]
    rsg_test.go:764: 5337 executions, 983 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn509228950
--- FAIL: TestRandomSyntaxSchemaChangeColumn (357.46s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928477&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928477&tab=artifacts#/) on master @ [fd2ab675441d1928650a0e3ed7eb3aa0395f51b2](https://github.com/cockroachdb/cockroach/commits/fd2ab675441d1928650a0e3ed7eb3aa0395f51b2):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8660354 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance.(*Instance).heartbeatLoop(0xc049395620, 0x5cadd60, 0xc0582d9100)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance/slinstance.go:190 +0x19e
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00d98a200, 0x5cade20, 0xc003202840, 0x0, 0xc01f11b660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8660279 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc011124190, 0x245c)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc011124180)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc010f460a0, 0x5cade20, 0xc02ffc73e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00d98a200, 0x5cade20, 0xc02ffc73e0, 0x0, 0xc0190edc90)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8660287 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc011124190, 0x2442)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc011124180)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc010f460a0, 0x5cade20, 0xc02ffc73e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00d98a200, 0x5cade20, 0xc02ffc73e0, 0x0, 0xc0190edd10)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN ident CHAR VARYING ( 66 ) ARRAY
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN ident CHAR VARYING ( 66 ) ARRAY". currently executing: map[ALTER TABLE ident.ident ADD COLUMN ident CHAR VARYING ( 66 ) ARRAY:1]
    rsg_test.go:764: 4414 executions, 809 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn282009305
--- FAIL: TestRandomSyntaxSchemaChangeColumn (353.71s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928635&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928635&tab=artifacts#/) on master @ [9ae5896d1944df379cc8005d597d0e89d710da81](https://github.com/cockroachdb/cockroach/commits/9ae5896d1944df379cc8005d597d0e89d710da81):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8779164 [chan send, 1 minutes]:
        github.com/lib/pq.(*conn).watchCancel.func1(0xc00d159bc0, 0xc00344d600, 0xc037288c60)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:104 +0x185
        created by github.com/lib/pq.(*conn).watchCancel
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:93 +0xa5
        
        goroutine 8697047 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5caeec0, 0xc020062600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0198c2200, 0x5caeec0, 0xc020062600, 0x0, 0xc015f2c510)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8697092 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc00cf9a350, 0x4998)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc00cf9a340)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc03e4983c0, 0x5caeec0, 0xc017003620)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0198c2200, 0x5caeec0, 0xc017003620, 0x0, 0xc02791bd30)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8697051 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5caeec0, 0xc020062600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0198c2200, 0x5caeec0, 0xc020062600, 0x0, 0xc015f2c550)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP COLUMN IF EXISTS IF RESTRICT
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP COLUMN IF EXISTS IF RESTRICT". currently executing: map[ALTER TABLE ident.ident DROP COLUMN IF EXISTS IF RESTRICT:1]
    rsg_test.go:764: 9432 executions, 1787 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn049238292
--- FAIL: TestRandomSyntaxSchemaChangeColumn (358.80s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928761&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928761&tab=artifacts#/) on master @ [9f198e190812bf301e8a2126cf43cd78e1cfb163](https://github.com/cockroachdb/cockroach/commits/9f198e190812bf301e8a2126cf43cd78e1cfb163):


```
        	/usr/local/go/src/crypto/tls/conn.go:779 +0x62
        bytes.(*Buffer).ReadFrom(0xc00ea04d00, 0x5c06ec0, 0xc01780cba0, 0x4502a5, 0x476ae00, 0x4d179e0)
        	/usr/local/go/src/bytes/buffer.go:204 +0xb1
        crypto/tls.(*Conn).readFromUntil(0xc00ea04a80, 0x5c102e0, 0xc0246e5d58, 0x5, 0xc0246e5d58, 0xc01c769301)
        	/usr/local/go/src/crypto/tls/conn.go:801 +0xf3
        crypto/tls.(*Conn).readRecordOrCCS(0xc00ea04a80, 0x0, 0x0, 0x6e85d1)
        	/usr/local/go/src/crypto/tls/conn.go:608 +0x115
        crypto/tls.(*Conn).readRecord(...)
        	/usr/local/go/src/crypto/tls/conn.go:576
        crypto/tls.(*Conn).Read(0xc00ea04a80, 0xc00518a000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        	/usr/local/go/src/crypto/tls/conn.go:1252 +0x15f
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*readTimeoutConn).Read(0xc0087cb640, 0xc00518a000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:1675 +0x129
        bufio.(*Reader).fill(0xc00e8a0948)
        	/usr/local/go/src/bufio/bufio.go:101 +0x105
        bufio.(*Reader).ReadByte(0xc00e8a0948, 0x2a46ae59, 0xed81a4d63, 0x0)
        	/usr/local/go/src/bufio/bufio.go:253 +0x39
        github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase.(*ReadBuffer).ReadTypedMsg(0xc00e8a1108, 0x5c679c0, 0xc00e8a0948, 0x5c5d100, 0x81da218, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase/encoding.go:203 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl.func4(0xc00e8a0900, 0xc00e236a40, 0xc000d18100, 0xc0017277cd, 0x5c7a440, 0xc01329c680, 0xc001727830, 0xc0017277cc, 0xc011870f00, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:328 +0x71
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl(0xc00e8a0900, 0x5caf080, 0xc009273f80, 0xc00e236a30, 0xc000d18100, 0x5400, 0x15000, 0xc012383d60, 0x0, 0x4, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:459 +0x565
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveConn(0xc01f2dbf00, 0x5caf080, 0xc009273f80, 0x5d16be0, 0xc00ea04a80, 0xc00b03a02b, 0x4, 0xc009273f50, 0x5c5a8c0, 0xc01425abd0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:152 +0x1b8
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc01f2dbf00, 0x5caf080, 0xc009273f80, 0x5d1af60, 0xc0246e5d58, 0x5caf001, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:674 +0xdd8
        github.com/cockroachdb/cockroach/pkg/server.(*SQLServer).startServeSQL.func1.1(0x5d1af60, 0xc0246e5d58)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2096 +0xf7
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc01113a280, 0x5caf080, 0xc026564600, 0xc01bdd8f70, 0x5d1af60, 0xc0246e5d58, 0xc0137454a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:156 +0xaf
        created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:153 +0x24a
        
        goroutine 8609339 [select, 5 minutes]:
        google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc02d8c2580)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:69 +0xc8
        created by google.golang.org/grpc.newCCBalancerWrapper
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:60 +0x172
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER FORCE_INDEX SET NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER FORCE_INDEX SET NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER FORCE_INDEX SET NOT NULL:1]
    rsg_test.go:764: 9983 executions, 1914 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn074417203
--- FAIL: TestRandomSyntaxSchemaChangeColumn (349.96s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928833&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928833&tab=artifacts#/) on master @ [9f198e190812bf301e8a2126cf43cd78e1cfb163](https://github.com/cockroachdb/cockroach/commits/9f198e190812bf301e8a2126cf43cd78e1cfb163):


```
        	/usr/local/go/src/io/io.go:314 +0x87
        io.ReadFull(...)
        	/usr/local/go/src/io/io.go:333
        golang.org/x/net/http2.readFrameHeader(0xc0124cba78, 0x9, 0x9, 0x5c06c60, 0xc00beb8180, 0x0, 0x0, 0xc030723c50, 0x5800000000)
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x89
        golang.org/x/net/http2.(*Framer).ReadFrame(0xc0124cba40, 0xc030723c50, 0x8181260, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa5
        google.golang.org/grpc/internal/transport.(*http2Server).HandleStreams(0xc0111f8900, 0xc030723da0, 0x55202c0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/internal/transport/http2_server.go:459 +0x98
        google.golang.org/grpc.(*Server).serveStreams(0xc016e88000, 0x5d1d840, 0xc0111f8900)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:827 +0xfe
        google.golang.org/grpc.(*Server).handleRawConn.func1(0xc016e88000, 0x5d1d840, 0xc0111f8900)
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:786 +0x3f
        created by google.golang.org/grpc.(*Server).handleRawConn
        	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:785 +0x52b
        
        goroutine 9059418 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).run(0xc004413a20, 0x5caf080, 0xc0164350b0, 0xc023463c40, 0xc01588a300)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:238 +0x438
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).Start.func1(0x5caf080, 0xc0164350b0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:196 +0x50
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc01588a300, 0x5caf080, 0xc0164350b0, 0x0, 0xc023463c60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9058513 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc014b14450, 0x393b)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc014b14440)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0184eebe0, 0x5caf080, 0xc016f1e120)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc01588a300, 0x5caf080, 0xc016f1e120, 0x0, 0xc001d0aef0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD ident STRING ( 424 ) ARRAY
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD ident STRING ( 424 ) ARRAY". currently executing: map[ALTER TABLE ident.ident ADD ident STRING ( 424 ) ARRAY:1]
    rsg_test.go:764: 7379 executions, 1361 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn151167974
--- FAIL: TestRandomSyntaxSchemaChangeColumn (354.18s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928960&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928960&tab=artifacts#/) on master @ [9f198e190812bf301e8a2126cf43cd78e1cfb163](https://github.com/cockroachdb/cockroach/commits/9f198e190812bf301e8a2126cf43cd78e1cfb163):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8456298 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run(0xc00a9a09c0, 0x5caf080, 0xc00592a660, 0xc0340e2300)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:121 +0x2b3
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).Start.func1(0x5caf080, 0xc00592a660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:100 +0x47
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0340e2300, 0x5caf080, 0xc00592a660, 0x0, 0xc014858500)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8456232 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc01476a190, 0x1e17)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc01476a180)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0144905a0, 0x5caf080, 0xc013c51230)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0340e2300, 0x5caf080, 0xc013c51230, 0x0, 0xc002355cf0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8456271 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc01476a190, 0x1e00)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc01476a180)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0144905a0, 0x5caf080, 0xc013c51230)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0340e2300, 0x5caf080, 0xc013c51230, 0x0, 0xc002355f60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP COLUMN FILES CASCADE
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP COLUMN FILES CASCADE". currently executing: map[ALTER TABLE ident.ident DROP COLUMN FILES CASCADE:1]
    rsg_test.go:764: 3608 executions, 632 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn112169867
--- FAIL: TestRandomSyntaxSchemaChangeColumn (359.96s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929186&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929186&tab=artifacts#/) on master @ [7d55af4a45bccf631c5afe1b7466523453e7cef8](https://github.com/cockroachdb/cockroach/commits/7d55af4a45bccf631c5afe1b7466523453e7cef8):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8712283 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x5caf080, 0xc016dbc660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:103 +0x13d
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc033034200, 0x5caf080, 0xc016dbc660, 0x0, 0xc022428e90)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8712216 [IO wait, 5 minutes]:
        internal/poll.runtime_pollWait(0x7f005826b2e8, 0x72, 0x0)
        	/usr/local/go/src/runtime/netpoll.go:222 +0x55
        internal/poll.(*pollDesc).wait(0xc0379fd998, 0x72, 0x0, 0x0, 0x4e1245c)
        	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
        internal/poll.(*pollDesc).waitRead(...)
        	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
        internal/poll.(*FD).Accept(0xc0379fd980, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        	/usr/local/go/src/internal/poll/fd_unix.go:394 +0x1fc
        net.(*netFD).accept(0xc0379fd980, 0x37e11d600, 0x0, 0x0)
        	/usr/local/go/src/net/fd_unix.go:172 +0x45
        net.(*TCPListener).accept(0xc0374f21a0, 0xc0000abe10, 0xc0000abe18, 0x28)
        	/usr/local/go/src/net/tcpsock_posix.go:139 +0x32
        net.(*TCPListener).Accept(0xc0374f21a0, 0x5518400, 0xc0235fcc00, 0x5d1af60, 0xc03633dd00)
        	/usr/local/go/src/net/tcpsock.go:261 +0x65
        github.com/cockroachdb/cmux.(*cMux).Serve(0xc0235fcc00, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:124 +0xa4
        github.com/cockroachdb/cockroach/pkg/server.(*Server).startListenRPCAndSQL.func5.2.1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:1999 +0x2f
        sync.(*Once).doSlow(0xc0302c4600, 0xc000099720)
        	/usr/local/go/src/sync/once.go:66 +0xec
        sync.(*Once).Do(...)
        	/usr/local/go/src/sync/once.go:57
        github.com/cockroachdb/cockroach/pkg/server.(*Server).startListenRPCAndSQL.func5.2(0x5caf080, 0xc016dbc660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:1998 +0x74
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc033034200, 0x5caf080, 0xc016dbc660, 0x0, 0xc023ffa7e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL". currently executing: map[ALTER TABLE ident.ident ALTER COLUMN ident SET NOT NULL:1]
    rsg_test.go:764: 8174 executions, 1519 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn977925304
--- FAIL: TestRandomSyntaxSchemaChangeColumn (356.02s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932922&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932922&tab=artifacts#/) on master @ [852e22a2ad78b1a19bd4f504f7079274c0c95fe5](https://github.com/cockroachdb/cockroach/commits/852e22a2ad78b1a19bd4f504f7079274c0c95fe5):


```
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc00baba100)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc03c242140, 0x5cb3460, 0xc02ff75800)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc036616280, 0x5cb3460, 0xc02ff75800, 0x0, 0xc00a101c40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9065616 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/jobs.(*jobScheduler).runDaemon.func1(0x5cb3460, 0xc016536660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/jobs/job_scheduler.go:364 +0x1e5
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc036616280, 0x5cb3460, 0xc016536660, 0x0, 0xc00abfc280)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9064746 [sync.Cond.Wait, 1 minutes]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc00baba110, 0x22d1)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc00baba100)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc03c242140, 0x5cb3460, 0xc02ff75800)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc036616280, 0x5cb3460, 0xc02ff75800, 0x0, 0xc00a101ca0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 9064847 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slstorage.(*Storage).deleteSessionsLoop(0xc02c7ba700, 0x5cb3460, 0xc016536660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slstorage/slstorage.go:283 +0x205
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc036616280, 0x5cb3460, 0xc016536660, 0x0, 0xc027d3ab80)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP COLUMN ident
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP COLUMN ident". currently executing: map[ALTER TABLE ident.ident DROP COLUMN ident:1]
    rsg_test.go:764: 4655 executions, 827 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn276816517
--- FAIL: TestRandomSyntaxSchemaChangeColumn (356.91s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2937582&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2937582&tab=artifacts#/) on master @ [445175daed0848429219755703febfb3b50a73e9](https://github.com/cockroachdb/cockroach/commits/445175daed0848429219755703febfb3b50a73e9):


```
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
        goroutine 8190376 [chan send]:
        github.com/lib/pq.(*conn).watchCancel.func1(0xc01429d560, 0xc00169bb80, 0xc00bc7b5c0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:104 +0x185
        created by github.com/lib/pq.(*conn).watchCancel
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:93 +0xa5
        
        goroutine 8127899 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc0125f0110, 0x31d2)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc0125f0100)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc0117a0140, 0x5cb6e60, 0xc01242ce10)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f9ea200, 0x5cb6e60, 0xc01242ce10, 0x0, 0xc005091b00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8127711 [select, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*baseQueue).processLoop.func2(0x5cb6e60, 0xc01242ca20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/queue.go:818 +0x15f
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f9ea200, 0x5cb6e60, 0xc01242ca20, 0x0, 0xc02a57d4c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8127807 [chan receive, 5 minutes]:
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop.func1(0x5cb6e60, 0xc01286c420)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:176 +0x76
        runtime/pprof.Do(0x5cb6e60, 0xc01286c420, 0xc000586e60, 0x1, 0x1, 0xc001d47fb8)
        	/usr/local/go/src/runtime/pprof/runtime.go:40 +0xcc
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop(0xc00d9bb300)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:175 +0x89
        created by github.com/cockroachdb/pebble.(*tableCacheShard).init
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident VARBIT ARRAY
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident VARBIT ARRAY". currently executing: map[ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS ident VARBIT ARRAY:1]
    rsg_test.go:764: 6358 executions, 1190 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn434893108
--- FAIL: TestRandomSyntaxSchemaChangeColumn (351.97s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-queries @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941632&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941632&tab=artifacts#/) on master @ [42e2e4265087751695afc05fc5ad8ec534fe0121](https://github.com/cockroachdb/cockroach/commits/42e2e4265087751695afc05fc5ad8ec534fe0121):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8805284 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/stats.(*Refresher).Start.func1(0x5cbb7a0, 0xc000074120)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/stats/automatic_stats.go:264 +0x1b1
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0110d0380, 0x5cbb7a0, 0xc000074120, 0x0, 0xc015e1ae60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8805137 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc014f94250, 0x3ff4)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc014f94240)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc013d70dc0, 0x5cbb820, 0xc014eaa660)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0110d0380, 0x5cbb820, 0xc014eaa660, 0x0, 0xc0148d1ee0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8805301 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).run(0xc00e644420, 0x5cbb820, 0xc01349f4a0, 0xc01610d0c0, 0xc0110d0380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:238 +0x438
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).Start.func1(0x5cbb820, 0xc01349f4a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:196 +0x50
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc0110d0380, 0x5cbb820, 0xc01349f4a0, 0x0, 0xc01610d0e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8877798 [chan send]:
        github.com/lib/pq.(*conn).watchCancel.func1(0xc015418720, 0xc001b2e840, 0xc00efd2480)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:104 +0x185
        created by github.com/lib/pq.(*conn).watchCancel
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:93 +0xa5
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP VARCHAR CASCADE
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP VARCHAR CASCADE". currently executing: map[ALTER TABLE ident.ident DROP VARCHAR CASCADE:1]
    rsg_test.go:764: 8458 executions, 1490 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn767114717
--- FAIL: TestRandomSyntaxSchemaChangeColumn (357.65s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-queries @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2944551&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2944551&tab=artifacts#/) on master @ [684e753c15f3fc58df79b6ea70e7b6715eae4835](https://github.com/cockroachdb/cockroach/commits/684e753c15f3fc58df79b6ea70e7b6715eae4835):


```
        github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase.(*ReadBuffer).ReadTypedMsg(0xc0018aec08, 0x5c7ada0, 0xc0018ae448, 0x5c704e0, 0x81f43c8, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase/encoding.go:203 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl.func4(0xc0018ae400, 0xc007cbad00, 0xc000ca5600, 0xc0017537cd, 0x5c8d860, 0xc016ac5480, 0xc001753830, 0xc0017537cc, 0xc00b76d200, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:328 +0x71
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl(0xc0018ae400, 0x5cc2560, 0xc01664e420, 0xc007cbacf0, 0xc000ca5600, 0x5400, 0x15000, 0xc01223f680, 0x0, 0x4, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:459 +0x565
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveConn(0xc002e07000, 0x5cc2560, 0xc01664e420, 0x5d2a1e0, 0xc0022dd180, 0xc004a69055, 0x4, 0xc01664e3f0, 0x5c6dca0, 0xc03324b980, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:152 +0x1b8
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc002e07000, 0x5cc2560, 0xc01664e420, 0x5d2e560, 0xc0017db270, 0x5cc2501, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:674 +0xdd8
        github.com/cockroachdb/cockroach/pkg/server.(*SQLServer).startServeSQL.func1.1(0x5d2e560, 0xc0017db270)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2096 +0xf7
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc00f87c200, 0x5cc2560, 0xc02c8e8600, 0xc0017dabb8, 0x5d2e560, 0xc0017db270, 0xc017bb00c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:156 +0xaf
        created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:153 +0x24a
        
        goroutine 8997332 [semacquire, 5 minutes]:
        sync.runtime_Semacquire(0xc013c08400)
        	/usr/local/go/src/runtime/sema.go:56 +0x45
        sync.(*WaitGroup).Wait(0xc013c083f8)
        	/usr/local/go/src/sync/waitgroup.go:130 +0x65
        golang.org/x/sync/errgroup.(*Group).Wait(0xc013c083f0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:40 +0x31
        github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.Wait(0xc013c083f0, 0x5cc24a0, 0xc00364e140, 0xc00e0b0420, 0xc00364e140)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:142 +0x57
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).RangeFeed(0xc005f68c00, 0x5cc2560, 0xc013c08390, 0xc014953380, 0x1, 0x8, 0xc014953388, 0x1, 0x1, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go:83 +0x3c7
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*dbAdapter).RangeFeed(0xc02cf64360, 0x5cc24a0, 0xc000ec5f40, 0xc014953380, 0x1, 0x8, 0xc014953388, 0x1, 0x1, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/db_adapter.go:69 +0xb4
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).run.func1(0x5cc24a0, 0xc000ec5f40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:208 +0xe3
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc00f87c200, 0x5cc24a0, 0xc000ec5f40, 0x0, 0xc013c08360)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident DROP IF EXISTS ident
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident DROP IF EXISTS ident". currently executing: map[ALTER TABLE ident.ident DROP IF EXISTS ident:1 ALTER TABLE ident.ident DROP METHOD CASCADE:1]
    rsg_test.go:731: 5m45s of 5m0s: 12758 executions, 2441 successful
    rsg_test.go:731: 5m50s of 5m0s: 12758 executions, 2441 successful
    rsg_test.go:731: 5m55s of 5m0s: 12758 executions, 2441 successful
    rsg_test.go:764: 12759 executions, 2442 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn384109106
--- FAIL: TestRandomSyntaxSchemaChangeColumn (359.30s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-queries @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945604&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945604&tab=artifacts#/) on master @ [dbf711a83969210bcf706d65f53f888856da6ad0](https://github.com/cockroachdb/cockroach/commits/dbf711a83969210bcf706d65f53f888856da6ad0):


```
        	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
        net.(*netFD).Read(0xc03e75f180, 0xc00f5a6800, 0x731, 0x731, 0x203002, 0x7fe5ab2a6058, 0x0)
        	/usr/local/go/src/net/fd_posix.go:55 +0x4f
        net.(*conn).Read(0xc02a92dd68, 0xc00f5a6800, 0x731, 0x731, 0x0, 0x0, 0x0)
        	/usr/local/go/src/net/net.go:182 +0x8e
        crypto/tls.(*atLeastReader).Read(0xc00b9be000, 0xc00f5a6800, 0x731, 0x731, 0x0, 0x5c267e0, 0xc01eac2f70)
        	/usr/local/go/src/crypto/tls/conn.go:779 +0x62
        bytes.(*Buffer).ReadFrom(0xc01897c280, 0x5c1d3c0, 0xc00b9be000, 0x4502a5, 0x477c060, 0x4d2a200)
        	/usr/local/go/src/bytes/buffer.go:204 +0xb1
        crypto/tls.(*Conn).readFromUntil(0xc01897c000, 0x5c26840, 0xc02a92dd68, 0x5, 0xc02a92dd68, 0xc011702001)
        	/usr/local/go/src/crypto/tls/conn.go:801 +0xf3
        crypto/tls.(*Conn).readRecordOrCCS(0xc01897c000, 0x0, 0x0, 0x6e85d1)
        	/usr/local/go/src/crypto/tls/conn.go:608 +0x115
        crypto/tls.(*Conn).readRecord(...)
        	/usr/local/go/src/crypto/tls/conn.go:576
        crypto/tls.(*Conn).Read(0xc01897c000, 0xc01a921000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        	/usr/local/go/src/crypto/tls/conn.go:1252 +0x15f
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*readTimeoutConn).Read(0xc01ec6cb80, 0xc01a921000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:1675 +0x129
        bufio.(*Reader).fill(0xc018b2c048)
        	/usr/local/go/src/bufio/bufio.go:101 +0x105
        bufio.(*Reader).ReadByte(0xc018b2c048, 0x2027da10, 0xed8223cd0, 0x0)
        	/usr/local/go/src/bufio/bufio.go:253 +0x39
        github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase.(*ReadBuffer).ReadTypedMsg(0xc018b2c808, 0x5c7e160, 0xc018b2c048, 0x5c738a0, 0x81f83e8, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwirebase/encoding.go:203 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl.func4(0xc018b2c000, 0xc02c631a70, 0xc01c396680, 0xc01eac37cd, 0x5c90c20, 0xc03ae49580, 0xc01eac3830, 0xc01eac37cc, 0xc042a88300, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:328 +0x71
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl(0xc018b2c000, 0x5cc5960, 0xc0294499b0, 0xc02c631a60, 0xc01c396680, 0x5400, 0x15000, 0xc0314379a0, 0x0, 0x4, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:459 +0x565
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveConn(0xc02763cc00, 0x5cc5960, 0xc0294499b0, 0x5d2d5e0, 0xc01897c000, 0xc01abbf042, 0x4, 0xc029449980, 0x5c71060, 0xc024be7e30, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:152 +0x1b8
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc02763cc00, 0x5cc5960, 0xc0294499b0, 0x5d31960, 0xc02a92dd68, 0x5cc5901, 0x0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:674 +0xdd8
        github.com/cockroachdb/cockroach/pkg/server.(*SQLServer).startServeSQL.func1.1(0x5d31960, 0xc02a92dd68)
        	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:2096 +0xf7
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc036828200, 0x5cc5960, 0xc022ff0720, 0xc00c901718, 0x5d31960, 0xc02a92dd68, 0xc00ed8fe30)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:156 +0xaf
        created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:153 +0x24a
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS IFNULL TIMETZ FAMILY ident
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS IFNULL TIMETZ FAMILY ident". currently executing: map[ALTER TABLE ident.ident ADD COLUMN IF NOT EXISTS IFNULL TIMETZ FAMILY ident:1]
    rsg_test.go:764: 3988 executions, 738 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn070299382
--- FAIL: TestRandomSyntaxSchemaChangeColumn (350.96s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-queries @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSchemaChangeColumn [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949835&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949835&tab=artifacts#/) on master @ [f9dca40d3afb0a605eeee76bec4a07ff65d6fec0](https://github.com/cockroachdb/cockroach/commits/f9dca40d3afb0a605eeee76bec4a07ff65d6fec0):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8797640 [chan receive, 5 minutes]:
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop.func1(0x5ccc2c0, 0xc011c56030)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:176 +0x76
        runtime/pprof.Do(0x5ccc2c0, 0xc011c56030, 0xc00000ff40, 0x1, 0x1, 0xc004baefb8)
        	/usr/local/go/src/runtime/pprof/runtime.go:40 +0xcc
        github.com/cockroachdb/pebble.(*tableCacheShard).releaseLoop(0xc00fcc6300)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:175 +0x89
        created by github.com/cockroachdb/pebble.(*tableCacheShard).init
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/table_cache.go:167 +0x1a5
        
        goroutine 8797658 [chan receive, 5 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).Start.func1(0x5ccc2c0, 0xc014280e10)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:193 +0x4c
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc012506300, 0x5ccc2c0, 0xc014280e10, 0x0, 0xc019ae03a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
        goroutine 8850920 [chan send, 1 minutes]:
        github.com/lib/pq.(*conn).watchCancel.func1(0xc027c84d80, 0xc0021ca000, 0xc00c1ae720)
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:104 +0x185
        created by github.com/lib/pq.(*conn).watchCancel
        	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/lib/pq/conn_go18.go:93 +0xa5
        
        goroutine 8797770 [sync.Cond.Wait]:
        runtime.goparkunlock(...)
        	/usr/local/go/src/runtime/proc.go:312
        sync.runtime_notifyListWait(0xc013df21d0, 0x2c4d)
        	/usr/local/go/src/runtime/sema.go:513 +0xf8
        sync.(*Cond).Wait(0xc013df21c0)
        	/usr/local/go/src/sync/cond.go:56 +0x9d
        github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc012f321e0, 0x5ccc2c0, 0xc014280e10)
        	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:251 +0x8b
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc012506300, 0x5ccc2c0, 0xc014280e10, 0x0, 0xc05432fd60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc
        
    rsg_test.go:186: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:755: Crash detected: 
        ALTER TABLE ident.ident ADD COLUMN PRECISION STRING ( 202 ) ARRAY [ 209 ]
        
        Stack trace:
        timeout: "ALTER TABLE ident.ident ADD COLUMN PRECISION STRING ( 202 ) ARRAY [ 209 ]". currently executing: map[ALTER TABLE ident.ident ADD COLUMN PRECISION STRING ( 202 ) ARRAY [ 209 ]:1]
    rsg_test.go:764: 5706 executions, 1045 successful
    rsg_test.go:401: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRandomSyntaxSchemaChangeColumn489812988
--- FAIL: TestRandomSyntaxSchemaChangeColumn (345.88s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestRandomSyntaxSchemaChangeColumn PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63944 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-21.1]
- #63202 sql/tests: TestRandomSyntaxSchemaChangeColumn failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-queries @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSchemaChangeColumn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/tests/rsg_test.go
+++ b/pkg/sql/tests/rsg_test.go
@@ -48,9 +48,10 @@ import (
 )
 
 var (
-	flagRSGTime        = flag.Duration("rsg", 0, "random syntax generator test duration")
-	flagRSGGoRoutines  = flag.Int("rsg-routines", 1, "number of Go routines executing random statements in each RSG test")
-	flagRSGExecTimeout = flag.Duration("rsg-exec-timeout", 15*time.Second, "timeout duration when executing a statement")
+	flagRSGTime                    = flag.Duration("rsg", 0, "random syntax generator test duration")
+	flagRSGGoRoutines              = flag.Int("rsg-routines", 1, "number of Go routines executing random statements in each RSG test")
+	flagRSGExecTimeout             = flag.Duration("rsg-exec-timeout", 15*time.Second, "timeout duration when executing a statement")
+	flagRSGExecColumnChangeTimeout = flag.Duration("rsg-exec-column-change-timeout", 2*time.Minute, "timeout duration when executing a statement for random column changes")
 )
 
 func verifyFormat(sql string) error {
@@ -120,8 +121,12 @@ type nonCrasher struct {
 func (c *nonCrasher) Error() string {
 	return c.err.Error()
 }
-
 func (db *verifyFormatDB) exec(t *testing.T, ctx context.Context, sql string) error {
+	return db.execWithTimeout(t, ctx, sql, *flagRSGExecTimeout)
+}
+func (db *verifyFormatDB) execWithTimeout(
+	t *testing.T, ctx context.Context, sql string, duration time.Duration,
+) error {
 	if err := verifyFormat(sql); err != nil {
 		db.verifyFormatErr = err
 		return err
@@ -158,7 +163,7 @@ func (db *verifyFormatDB) exec(t *testing.T, ctx context.Context, sql string) er
 			return &nonCrasher{sql: sql, err: err}
 		}
 		return nil
-	case <-time.After(*flagRSGExecTimeout):
+	case <-time.After(duration):
 		db.mu.Lock()
 		defer db.mu.Unlock()
 		b := make([]byte, 1024*1024)
@@ -396,7 +401,7 @@ func TestRandomSyntaxSchemaChangeColumn(t *testing.T) {
 	}, func(ctx context.Context, db *verifyFormatDB, r *rsg.RSG) error {
 		n := r.Intn(len(roots))
 		s := fmt.Sprintf("ALTER TABLE ident.ident %s", r.Generate(roots[n], 500))
-		return db.exec(t, ctx, s)
+		return db.execWithTimeout(t, ctx, s, *flagRSGExecColumnChangeTimeout)
 	})
 }
 

